counsel.el

```diff
@@ -2669,7 +2669,8 @@ otherwise continue prompting for tags."
       (fset 'org-set-tags store))))
 
 (defcustom counsel-org-goto-display-style 'path
-  "The style for displaying headlines in `counsel-org-goto' functions.
+  "The style for displaying headlines in `counsel-org-goto' and \
+`counsel-org-agenda-headlines' functions.
```

Comment:
Please follow the conventions outlined in [`(elisp) Documentation Tips`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html), in particular keeping the first line of docstrings brief and to the point, e.g.

```
  "The style for displaying Org headlines.

This is used by functions `counsel-org-goto' and
`counsel-org-agenda-headlines'.
```

or similar. The basic idea is not to list call sites in the variable's summary. The same applies to the other docstrings in this patch.